Use newest Firefox for integration tests with geckodriver

oliverguenther opened the pull request.

oliverguenther committed:
Use Firefox latest on travis.yml

ulferts commented on spec/support/capybara.rb:
```diff
@@ -44,7 +44,7 @@
 
   # need to disable marionette as noted
   # https://github.com/teamcapybara/capybara#capybara
```
Please remove the comment

ulferts commented on .travis.yml:
```diff
@@ -107,7 +113,7 @@ notifications:
 
 addons:
-  firefox: "45.0esr"
+  firefox: "latest"
```
why not the latest `esr` as this is what we officially support?

oliverguenther replied:
Latest ESR is 52, too old for testing with Marionette (things like double_click , send_keys will not work properly).

oliverguenther committed:
Pass profile through options to avoid deprecation

oliverguenther committed:
Scroll to element when blocked by overflow

The details view blocks certain elements of the table, causing
clicks to no longer work in newer Firefoxen

This is mainly caused by
https://bugzilla.mozilla.org/show_bug.cgi?id=1295538

oliverguenther committed:
Send keys to correct element

oliverguenther committed:
Extend documentation

oliverguenther committed:
Don't use evaluate_script

oliverguenther committed:
Use marionette with cucumber

oliverguenther committed:
Scroll to elements before interacting

oliverguenther committed:
Replace or remove more evaluate_scripts

oliverguenther committed:
Try latest beta

oliverguenther committed:
Use headless firefox

oliverguenther committed:
Document use of FF 57

oliverguenther committed:
Add headless mode using FF 57

oliverguenther committed:
Fix cukes

oliverguenther committed:
Resize selenium window before specs

oliverguenther committed:
Bump rspec-retry and reset Capybara between retries

oliverguenther committed:
Wait for animations in ngdialog to complete

ulferts committed:
remove cukes inactive for 3 years

[ci skip]

ulferts committed:
fix typo

[ci skip]

ulferts commented:
Works fine. I fixed a typo.

I also piggybacked the PR to remove cukes that have been deactivated three years ago.
